### PR TITLE
Use none bootloader in case of nfs root boot filesystem

### PR DESCRIPTION
**Pull request by teclator**
[Trello Card](https://trello.com/c/siYsznJj/1433-3-help-with-nfs-root-in-mbrgap-bug)

Tested manually

nfsrootbootloader

### CODE REVIEW

**Comment by ancorgs** [comment]
Shouldn't this go to the SLE-15-GA branch first? We want it in the media for sure

File: src/lib/bootloader/finish_client.rb [hunk 1]
```diff
@@ -44,6 +44,7 @@ def write
 
       # we do not manage bootloader, so relax :)
       return true if bl_current.name == "none"
+      return true if Yast::BootStorage.boot_filesystem.is?(:nfs)
```
Comment (ancorgs) [inline src/lib/bootloader/finish_client.rb]: Isn't this somehow redundant with the previous condition? I mean, if `boot_filesystem.is?(:nfs)` couldn't we be quite sure than `bl_current.name == "none"` (according to `ProposalClient#make_proposal`)

Comment (teclator) [reply src/lib/bootloader/finish_client.rb]: It should be redundant, it was the first defensive check I added, before proposing a "none" bootloader.

**Comment by teclator** [comment]
@ancorgs modified branch and removed the redundant check.

File: src/modules/Bootloader.rb [hunk 2]
```diff
@@ -204,7 +204,8 @@ def Summary(simple_mode: false)
       # kokso: additional warning that root partition is nfs type -> bootloader will not be installed
       if BootStorage.boot_filesystem.is?(:nfs)
         log.info "Bootloader::Summary() -> Boot partition is nfs type, bootloader will not be installed."
-        return _("The boot partition is of type NFS. Bootloader cannot be installed.")
+        msg = _("The boot partition is of type NFS. Bootloader cannot be installed.")
+        return (simple_mode ? [msg] : [Yast::HTML.Colorize(msg, "red")])
```
Comment (jreidinger) [inline src/modules/Bootloader.rb]: why having it red? It is not warning. It is correct proposal for NFS.

File: src/modules/Bootloader.rb [hunk 3]
```diff
@@ -204,7 +204,8 @@ def Summary(simple_mode: false)
       # kokso: additional warning that root partition is nfs type -> bootloader will not be installed
       if BootStorage.boot_filesystem.is?(:nfs)
         log.info "Bootloader::Summary() -> Boot partition is nfs type, bootloader will not be installed."
-        return _("The boot partition is of type NFS. Bootloader cannot be installed.")
+        msg = _("The boot partition is of type NFS. Bootloader cannot be installed.")
```
Comment (jreidinger) [inline src/modules/Bootloader.rb]: I would rephrase it like `"The boot partition is of type NFS. No bootloader is installed."` Or if you want to have it red and with warning, then I would write something like `"The boot partition is of type NFS. No bootloader is installed. Please ensure that your machine can properly boot from network."` So at least user have hint, but for me it is not needed, as if user want nfs root ( not trivial to setup ), then he knows what he is doing.

File: src/modules/Bootloader.rb (hunk 2 above)
Comment (teclator) [reply src/modules/Bootloader.rb]: I agree with no highlight but definitely should be shown. I just used the same warning was defined in the NoneBootloader [Summary](https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/none_bootloader.rb#L11)

File: src/modules/Bootloader.rb (hunk 3 above)
Comment (teclator) [reply src/modules/Bootloader.rb]: Just used the same message, but also fine about rewording it.

Comment (ancorgs) [reply src/modules/Bootloader.rb]: If the original message is already translated, then I wouldn't change it
